fastlane/lib/fastlane/actions/commit_version_bump.rb

```diff
@@ -227,6 +275,16 @@ def settings_bundle_file_path(project, settings_file_name)
 
           File.join(settings_bundle.real_path, settings_file_name)
         end
+
+        def modified_files_relative_to_repo_root(repo_root)
+          return [] if Actions.lane_context[SharedValues::MODIFIED_FILES].nil?
+
+          root_pathname = Pathname.new repo_root
+          Actions.lane_context[SharedValues::MODIFIED_FILES].map do |path|
+            next path unless path =~ %r{^/}
+            Pathname.new(path).relative_path_from(root_pathname).to_s
+          end.uniq
```

Comment:
Could we separate this out into another line just to make sure that it is totally clear whats being returned here?